generators/client/templates/angular/src/main/webapp/app/layouts/navbar/navbar.scss.ejs

```diff
@@ -42,21 +45,25 @@ Navbar
         background-color: $dark;
```

Comment:
all color variations in this stylesheet are jhipster custom theme specific. On choosing external theme, it would be good to not apply these as default